pkg/generator/generator_test.go

```diff
@@ -545,3 +547,22 @@ func TestGenBuild(t *testing.T) {
 		t.Errorf("\nTest failed. Below is the diff of the expected vs actual results.\nRed text is missing and green text is extra.\n\n" + dmp.DiffPrettyText(diffs))
 	}
 }
+
+func TestWriteFileAndPrint(t *testing.T) {
+	deployDir, err := ioutil.TempDir("", "test-write-file-and-print")
+	if err != nil {
+		t.Error(err)
+		return
+	}
+
+	defer os.RemoveAll(deployDir)
+
+	olmCatalogPackagePath := filepath.Join(deployDir, olmCatalogDir, catalogPackageYaml)
+	if err = writeFileAndPrint(olmCatalogPackagePath, []byte("sometext"), defaultFileMode); err != nil {
+		t.Error("Test failed. Failed to write file and print:", err)
```

Comment:
`t.Error`=>`t.Errorf`

Reply:
`"Test failed. Failed to write file and print:"` ->
`"failed to write file and print:"`?

also why `t.Error` but not `t.Fatalf`?  I believe `t.Fatalf` exits immediately so that you don't need `return` at line 563.

Reply:
Actually yes, t.Errorf() + return can be replaced by t.Fatalf. That will still run all deferred functions.